radar.csv

```diff
@@ -31,7 +31,8 @@
 "Keepass files in GitHub",hold,"Tools",true,"Use a password manager (or Vault) instead."
 "Docker",adopt,"Tools",true,"Docker should be used as the default containerisation technology. Used for building, testing and deployment."
 "gRPC",trial,"Tools",true,"<a href='https://grpc.io/'>gRPC</a> is a high-performance RPC framework which we may investigate for high-volume data collection (for example, SQL Monitor)."
-"WhiteSource (vulnerability scanning)",trial,"Tools",true,"We currently use WhiteSource for vulnerability scanning. This is currently under review."
+"WhiteSource (vulnerability scanning)",hold,"Tools",true,"We have tried using WhiteSource for vulnerability scanning but found that it didn't meet our needs."
```

Comment:
Should this be hold, or should we remove it altogether as we are actively moving away from it?

Reply:
Let's get rid of it completely (otherwise, it's confusing for new starters?).

Reply:
Done :)